app/views/katello/api/v2/subscriptions/base.json.rabl

```diff
@@ -0,0 +1,19 @@
+object @resource
+
+attributes :cp_id => :id
```

Comment:
Is this for backwards compatibility or another reason we don't want to make use of the database id?